packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -2940,4 +2940,25 @@ export default class Meeting extends StatelessWebexPlugin {
 
     return Promise.all(qualityActions);
   }
+
+  /**
+  * @param {Object} options parameter
+  * @param {Boolean} options.sendAudio send audio from the display share
+  * @param {Boolean} options.sendShare send video from the display share
+  * @param {Object} options.sharePreferences
+  * @param {MediaTrackConstraints} options.sharePreferences.shareConstraints constraints to apply to video #see https://developer.mozilla.org/en-US/docs/Web/API/MediaTrackConstraints
+  * @param {Boolean} options.sharePreferences.highFrameRate if shareConstraints isn't provided, set default values based off of this boolean
+  * @returns {Promise.<MediaStream>}
+  */
+  shareScreen(options = {sendShare: true, sendAudio: false}) {
+    LoggerProxy.logger.log('Meeting:index#shareScreen --> Getting local share');
+
+    return Media.getDisplayMedia(options).then((shareStream) => {
+      this.updateShare({
+        sendShare: true,
```

Comment:
Should this match the parameter passed in through the 'options' object?

Reply:
No. The naming of the params unfortunately collides and the `sendShare` that is sent to `getDisplayMedia` is not the same as the `sendShare` to `updateShare`. `updateShare` tells the meeting that we are about to share the stream passed in as `shareStream`.

Reply:
@lalli-flores there's plenty of developer friendly improvements that could be made to the meetings plugin, but outside the scope of this ticket